src/frontend/sharedComponents/BottomSheetModal/index.tsx

```diff
@@ -66,7 +66,7 @@ const useBackPressHandler = (onHardwareBackPress?: () => void) => {
 };
 
 const useSnapPointsCalculator = () => {
-  const [sheetHeight, setSheetHeight] = React.useState(0);
+  const [sheetHeight, setSheetHeight] = React.useState(-1);
```

Comment:
Probably should write a comment about the context for this cursed value 😅 can do as a follow up but for poserity:

Having a value of 0 caused a flickering of the backdrop whenever the sheet was initially opened. This only occurs when the component is used within a component (as opposed to a standalone navigation screen). Switching it to a positive number causes the `onLayout` event in the content to report an inaccurate height value, which causes the sheet size to be miscalculated, affecting the content within the sheet (e.g. squishing, overflow, etc).